### ISSUE TITLE
Swagger Version not defined and unable to render

### ISSUE THREAD

**Issue by SelfhostedPro** [nitrojs/nitro#2361]
### Environment

latest everything (node latest LTS)

### Reproduction

https://stackblitz.com/edit/github-ujxcqv?file=nitro.config.ts

### Describe the bug

Swagger docs aren't visible when going to /_nitro/swagger and pops up the following error: 
```
Unable to render this definition
The provided definition does not specify a valid version field.

Please indicate a valid Swagger or OpenAPI version field. Supported version fields are swagger: "2.0" and those that match openapi: 3.0.n (for example, openapi: 3.0.0).
```
<img width="375" alt="image" src="https://github.com/unjs/nitro/assets/66331933/1811dd5a-5738-4bc7-a035-1e84644e7e03">

**Comment by Kasparas-G55** [comment]
Came across the same issue, stable channel of nitro is currently using `swagger-ui-dist` v4 CDN, which doesn't support OpenAPI 3.1.0 which is what it is set to.

Switching to nightly channel should fix your problem.

**Comment by SelfhostedPro** [comment]
Ah, I thought I also ran into this on nightly. Is there a way to manually specify the OpenAI version in the nuxt config?

**Comment by Kasparas-G55** [comment]
Sadly it isn't possible since the value is hardcoded.
https://github.com/unjs/nitro/blob/3a69dd70bae87d12a6d9a86691fad48b2ebb044e/src/runtime/routes/openapi.ts#L25-L28
Though you could propose to use the upcoming meta object (https://github.com/unjs/nitro/pull/2299) to allow setting OpenAPI's version and use the latest if it isn't set, but I don't really see much purpose in that, would require more discussion.

**Comment by kspace-trk** [comment]
Hi, I expect this issue will be resolved when the version containing this Pull Request is released.
https://github.com/unjs/nitro/pull/2343

**Comment by pi0** [comment]
You can try with https://nitro.unjs.io/guide/nightly

**Comment by Exotelis** [comment]
This is absolutely fine, but please try to keep the documentation up to date. It becomes challenging to use features and contribute effectively if the docs are outdated.

**Comment by pi0** [comment]
Issue fixed in nitro 2.10+ releases. Note: Swagger UI is URL is now `/_swagger`
